src/shared/components/Button/Button.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+interface Button {
+  className: string;
+  onClick: React.MouseEventHandler<HTMLButtonElement>;
+  buttonText: string;
+}
+
+const Button: React.FC<Button> = ({
+  className,
+  onClick,
+  buttonText,
+}: Button) => {
+  return (
+    <button type="button" className={className} onClick={onClick}>
+      {buttonText}
+    </button>
+  );
+};
+
+export default Button;
```

Comment:
From now export all components with memoization:
`export default memo(Button)`
Read about memo - thanks to memoization component will not re-render if props don't change.

Reply:
Fixed